packages/flutter/lib/src/material/material_state.dart

```diff
@@ -168,6 +168,9 @@ abstract class MaterialStateColor extends Color implements MaterialStateProperty
   /// specified state.
   @override
   Color resolve(Set<MaterialState> states);
+
+  /// A constant whose value is [Colors.transparent] for all states.
+  static const MaterialStateColor transparent = _MaterialStateColorTransparent();
```

Comment:
Awesome!

Reply:
This doesn't change doesn't really belong in this PR, but I guess it can't hurt :-)